### ISSUE TITLE
The button to add a new folder to sync is inaccessible in Linux

### ISSUE THREAD

**Issue by vojtapolasek** [nextcloud/desktop#957]
### Expected behaviour
I should be able to focus the button for adding a new folder to sync in the main app window by pressing TAB key needed number of times.

### Actual behaviour
I can't focus the button. I can't even see it with flat review of Orca screenreader.

### Steps to reproduce
Open the NC Linux client and try to focus the button by pressing TAB key. It oes not work.
You can install and activate Orca screenreader to hear that it can't find the button, but simple visual check should be enough.

### Client configuration
Client version: 2.5.1

Operating system: Arch Linux

OS language: Czech

Qt version used by client package (Linux only, see also Settings dialog): 5.12.0

Client package (From Nextcloud or distro) (Linux only): Distro

Installation path of client: /usr

**Comment by camilasan** [comment]
Hi, Could you try to run the most recent desktop client and server to check if you still have this issue? The latest version can be seen by checking [https://nextcloud.com/install/#install-clients](https://nextcloud.com/install/#install-clients) and [https://nextcloud.com/install/#instructions-server](https://nextcloud.com/install/#instructions-server) respectively.

If you still have this problem with the latest desktop client, please re-open the issue.

Thanks!

**Comment by vojtapolasek** [comment]
Hello, well, I must say that the interface of the client changed, however, the accessibility did not improve. Or to be more precise:
Currently, I can go through the log of events and eventually click on "view more" and perform some action. I can open nextcloud apps in browser. But I can't move to another window, where settings used to be.
And moreover, the button to "add a new folder" is still not accesible.
Basically, what you can try is to open the client on Linux, use TAB key to move around elements. I hope you get a visual indication of it.

**Comment by vojtapolasek** [comment]
Can you please reopen? I am happy to test accessibility of NC client.

**Comment by camilasan** [comment]
> Can you please reopen? I am happy to test accessibility of NC client.

What about you create new github issues for the problems you are finding with the new version? Please use screenshots when talking about accessibility. It helps a lot to understand the issue.

**Comment by camilasan** [comment]
> I am happy to test accessibility of NC client.

Thank you ❤️
